ts/features/idpay/configuration/screens/InstrumentsEnrollmentScreen.tsx

```diff
@@ -129,33 +130,22 @@ export const InstrumentsEnrollmentScreen = () => {
       ),
       title: I18n.t("idpay.configuration.instruments.enrollmentSheet.header"),
       footer: (
-        <FooterWithButtons
-          type="TwoButtonsInlineThird"
-          primary={{
-            type: "Solid",
-            buttonProps: {
-              label: I18n.t(
-                "idpay.configuration.instruments.enrollmentSheet.buttons.activate"
-              ),
-              accessibilityLabel: I18n.t(
-                "idpay.configuration.instruments.enrollmentSheet.buttons.activate"
-              ),
-              onPress: handleEnrollConfirm
+        <FooterActionsInline
+          startAction={{
```

Comment:
As I observed in the previous files, the migration from  `FooterWithButtons` to `FooterActionsInline` requires mapping`primary` to `startAction` and `secondary` to `endAction`. 
In this case, did you swap them on purpose, or was it a mistake?